cmd/root.go

```diff
@@ -58,10 +63,30 @@ var rootCmd = &cobra.Command{
 		configs = append(configs, ocpMaster, ocpNode)
 
 		for _, config := range configs {
-			config.Fetch(outputDir)
-			config.Decode()
-			config.Translate()
-			ocp.DumpManifests(outputDir, config.GenYAML())
+			err := config.Fetch(outputDir)
+			if err != nil {
+				logrus.Fatal(err)
+			}
```

Comment:
same comment applies. If these are going to go to the user then more meaningful errors might be of use.